src/Admin/SiteHealth.php

```diff
@@ -257,11 +257,6 @@ public function add_debug_information( $debugging_information ) {
 							'value'   => AMP_Theme_Support::get_support_mode(),
 							'private' => false,
 						],
-						'amp_experiences_enabled' => [
```

Comment:
Site Health no longer shows the enabled experiences (since Website will be the only one).

Reply:
Good catch